src/types/types.ts

```diff
@@ -73,9 +70,10 @@ export type InstantSearchPagination = {
   nbPages: number
 }
 
+export type Facets = string | string[] | undefined
```

Comment:
Creation of own `Facets` type as the one from instant-search does not represent the reality.

`instant-search` may provide only a `string` as a `Facets` but its type states that it only accepts `string[] | undefined`